src/attackHandler.ts

```diff
@@ -0,0 +1,48 @@
+import { Character } from './character';
+import { WeaponCollision as collision } from './weaponCollision';
+
+export class AttackHandler {
+  static handleAttack(attacker: Character) {
+    if (attacker.weapon === undefined) {
+      return;
+    }
+    if (Date.now() - attacker.attackTimer >= attacker.weapon.cooldown) {
+      attacker.isAttacking = true;
```

Comment:
Move lines 10 and 11 to beginning of the next function.

Reply:
done